image/templates/helm/shared/templates/99-shared-openshift-monitoring.yaml

```
@@ -0,0 +1,80 @@
+{{- include "srox.init" . -}}
+{{- if ._rox.monitoring.openshift.enabled -}}
+  {{- if not ._rox.monitoring.openshift._securedWithCentral -}}
+---
+apiVersion: authorization.openshift.io/v1
+kind: ClusterRole
+metadata:
+  name: cluster-extension-apiserver-authentication-reader
```

We need the namespace here as well, right?
```suggestion
  name: cluster-extension-apiserver-authentication-reader-{{ .Release.Namespace }}
```

Or you could use this helper function https://github.com/stackrox/stackrox/blob/2f1ee951707e53ac4ed6fd503df94c510804c042/image/templates/helm/shared/templates/_metadata.tpl#L232.